cpp/src/parquet/arrow/reader.cc

```diff
@@ -1315,15 +1315,15 @@ Status FileReader::GetRecordBatchReader(const std::vector<int>& row_group_indice
   return Status::OK();
 }
 
-Status FileReader::Make(::arrow::MemoryPool* pool,
+Result<std::unique_ptr<FileReader>> Make(::arrow::MemoryPool* pool,
                         std::unique_ptr<ParquetFileReader> reader,
                         const ArrowReaderProperties& properties,
                         std::unique_ptr<FileReader>* out) {
```

Comment:
`std::unique_ptr<FileReader>* out` should be removed as part of the signature of the function. The Result already contains the `FileReader` value so this is unnecessary. See this commit with a similar change for an example:
https://github.com/apache/arrow/commit/0cdbdac413fa1e7acaf0362eb8ca8fd1911d2a9e

Reply:
Hey @raulcd updated this but I think i got something wrong on how to use the `std::unique_ptr<FileReader>* out` internally in the function.